test/expected/pg_dump.out

```diff
@@ -35,6 +36,33 @@ INSERT INTO "two_Partitions"("timeCustom", device_id, series_0, series_1) VALUES
 \set QUIET on
 \o
 \c :TEST_DBNAME :ROLE_SUPERUSER
+CREATE VIEW worker_counts AS
+WITH
+  tsdb_workers AS (SELECT * FROM pg_stat_activity WHERE application_name LIKE 'TimescaleDB%')
+SELECT count(*) FILTER (WHERE application_name LIKE '%Scheduler') AS scheduler_count,
+       count(*) FILTER (WHERE application_name LIKE '%Launcher') AS launcher_count
+  FROM tsdb_workers WHERE datname = :'TEST_DBNAME';
+CREATE FUNCTION wait_worker_counts(scheduler INTEGER, launcher INTEGER)
```

Comment:
Why parameters if you always pass zero for both? And IMHO we just need to check something like  `SELECT count(*) FROM pg_stat_activity WHERE datname = :'TEST_DBNAME';`

Reply:
It was generic code. I intended to put it in a separate file.

Reply:
> And IMHO we just need to check something like `SELECT count(*) FROM pg_stat_activity WHERE datname = :'TEST_DBNAME';`

The table `pg_stat_activity` contain other entries as well for applications that are not background workers, e.g., for the connected backend, so better to only look at the background workers.

Reply:
It turned out that it is not sufficient to wait for `pg_stat_activity` to remove the entry because that does not remove it from `procArray` in `procarray.c` quick enough, still reporting that it is connected. Instead, I added a separate function that checks `procArray` and waits for it to be clear, which means the process has exited.